### ISSUE TITLE
Ontobee shows different hierarchy than OWL file opened from URL

### ISSUE THREAD

**Issue by johnwjudkins** [ufbmi/OMRSE#140]
Example: Look at the page for 'patient discharge' in Ontobee, and go to the same class in Protégé after opening OMRSE from the URL. These show different parents for this class.

**Comment by CDowland** [comment]
OMRSE previously MIREOTed several classes from d-acts, but did not import d-acts as a whole. Several OMRSE terms made use of those d-acts terms in their axioms. Since then, many of those d-acts terms have changed, been replaced with similar terms, or simply deprecated. It was decided earlier this year to import d-acts into OMRSE, which meant that the OMRSE terms using outdated d-acts terms needed to be updated in order to reconcile OMRSE with the newer version of d-acts.

So, in the example you mention, ‘patient discharge’ was formerly a child of ‘document act,’ which was a child of ‘declaration,’ and the latter two are d-acts classes that have been replaced with the similar classes ‘deontic document act’ and ‘deontic declaration,’ respectively. Currently, ‘patient discharge’ is a child of ‘deontic document act,’ which is a child of ‘deontic declaration.’ 

As for the discrepancy with Ontobee, that is because Ontobee does not yet reflect these fairly recent changes. I am not sure how long it takes for that to happen.

**Comment by dillerm** [comment]
What was the proposed solution for this again?

**Comment by sjbost** [comment]
I don't think we had a proposed solution, just an idea that the problem might be caused by the fact that Ontobee doesn't reflect the most current version of d-acts. I believe you were going to investigate to determine what might be causing the issue.

**Comment by dillerm** [comment]
Ah, yes, I remember now. Thanks!

**Comment by CDowland** [comment]
I'm not sure whether this is helpful, but it might be:

Ontobee correctly reflects that 'identity datum' and its subclasses were deprecated/obsoleted, and it shows the new social identity ICE classes. 

However, it does not seem to show any changes more recent than that. For example, 'county smoking survey data' does not show up. Similarly, terms such as 'employee role' and 'student role' have been changed in OMRSE to no longer use SLGDC in their axioms. However, Ontobee still shows the old versions of them, with the old axioms. 

Just a few days after deprecating the 'identity datum' classes and adding the social identity ICEs in the devel branch, @dillerm did a merge. That was on June 25, 2020. So, the updates from that merge are reflected on Ontobee. But the changes since then do not seem to be, even though @dillerm did another merge in mid October, and I did another later in October.

In short, it seems that updates to the master branch that were made via the June merge are reflected on Ontobee, but not the ones from the October merges.

**Comment by dillerm** [comment]
Yeah, I just did a thorough vetting of the OMRSE direct imports to see if any of them were importing an old version IRI for omrse-core and everything checked out. I also checked the IRIs listed in the purl.obolibrary.org Github and everything seems to be fine there. I'm leaning toward this being an issue in Ontobee, itself. I can go ahead and write up an email to Oliver to let him know of this issue.

**Comment by dillerm** [comment]
I've updated the version info in omrse.owl and it looks like Ontobee picked up that change and updated the classes on the website.
